### PR TITLE
qrcp: renamed from qr-filetransfer, unstable-2018-10-22 -> 0.6.4

### PR DESCRIPTION
**Pull request by fgaz**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/networking/qrcp/default.nix [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    description = "Transfer files over wifi by scanning a QR code from your terminal";
+    longDescription = ''
+      qrcp binds a web server to the address of your Wi-Fi network
+      interface on a random port and creates a handler for it. The default
+      handler serves the content and exits the program when the transfer is
+      complete.
+    '';
+    license = licenses.mit;
+    maintainers = with maintainers; [ fgaz ];
+    platforms = platforms.all;
```
Comment (SuperSandro2000) [inline pkgs/tools/networking/qrcp/default.nix:32]: This is automatically set and the second final new line can be removed.

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -6523,7 +6523,9 @@ in
 
   qshowdiff = callPackage ../tools/text/qshowdiff { };
 
-  qr-filetransfer = callPackage ../tools/networking/qr-filetransfer { };
+  qrcp = callPackage ../tools/networking/qrcp { };
+
+  qr-filetransfer = throw ''"qr-filetransfer" has been renamed to "qrcp"'';
```
Comment (SuperSandro2000) [inline pkgs/top-level/all-packages.nix:6528]: This belongs into the alias.nix file.

File: pkgs/tools/networking/qrcp/default.nix (hunk 1 above)
Comment (fgaz) [reply pkgs/tools/networking/qrcp/default.nix:32]: Is it? from https://nixos.org/manual/nixpkgs/stable/index.html#sec-standard-meta-attributes

> If no platform is specified, the package does not have prebuilt binaries

Removed the newline.

File: pkgs/top-level/all-packages.nix (hunk 2 above)
Comment (fgaz) [reply pkgs/top-level/all-packages.nix]: Right, done.

**Comment by fgaz** [comment]
Rebased.

File: pkgs/tools/networking/qrcp/default.nix (hunk 1 above)
Comment (SuperSandro2000) [reply pkgs/tools/networking/qrcp/default.nix:32]: It is for buildGoModule.

Comment (fgaz) [reply pkgs/tools/networking/qrcp/default.nix]: Done

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 98494` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>qrcp</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 98494` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>qrcp</li>
  </ul>
</details>

**Pull request merged**